src/main/java/com/amazon/opendistroforelasticsearch/sql/query/maker/AggMaker.java

```diff
@@ -103,8 +103,9 @@ public AggregationBuilder makeGroupAgg(Field field) throws SqlParseException {
             }
             return makeRangeGroup(methodField);
         } else {
-            TermsAggregationBuilder termsBuilder = AggregationBuilders.terms(field.getName()).field(field.getName());
-            groupMap.put(field.getName(), new KVValue("KEY", termsBuilder));
+            String termName = (null != field.getAlias()) ? field.getAlias() : field.getName();
```

Comment:
Suggest to cover the empty string case by using Strings.isNullOrEmpty.

Reply:
sure, I'll change to use that.